src/components/product/AddToCart.tsx

```diff
@@ -0,0 +1,10 @@
+import { Products } from "../../types/Products";
+import "./addtocart.scss";
+
+export default function AddToCart(props:Products){
+  return (
+    <button className="btn  btn-outline-dark cart-btn">
```

Comment:
use linint tool, add as extension in your editor. also you can use husky pre commit and add a hook pretiier.

Reply:
ok